lib/animina_web/potential_partner.ex

```diff
@@ -52,6 +52,27 @@ defmodule AniminaWeb.PotentialPartner do
     |> Ash.read!()
   end
 
+  @doc """
+  We use this to fetch potential partners for the user during registration.
+  """
+
+  def potential_partners_on_registration(user) do
+    User
+    |> Ash.Query.for_read(:read)
+    |> Ash.Query.sort(Ash.Sort.expr_sort(fragment("RANDOM()")))
+    |> beta_registration_partner_gender_query(user)
```

Comment:
With this kind of naming we run into the "What do we do when this code moves from beta to normal?" are we doing a search and replace? Is that save? Would it be maybe better to just assume that this is not beta since we are working within this context?

Reply:
I have changed the naming of this